src/components/GameAbout/GameAbout.stories.tsx

```diff
@@ -1,42 +1,112 @@
-      '\n' +
-      'Pellentesque pellentesque enim quis mauris elementum sollicitudin. Sed pharetra ultrices posuere. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Suspendisse gravida laoreet mauris, quis hendrerit quam. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Morbi sit amet feugiat quam, quis feugiat justo.\n' +
-      '\n' +
-      'Duis tristique neque eu est tincidunt, ac sodales elit luctus. Praesent eget euismod leo. Phasellus in tortor aliquet, maximus felis sit amet, feugiat nisi. Nulla ornare porta rhoncus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Nulla mollis leo eu odio consectetur, a cursus nunc euismod. Sed porta leo lacus, sed tristique enim pharetra quis. Donec at nibh sit amet erat aliquet ullamcorper eu vitae dui.'
+// With Stickers and Button
+export const WithStickersAndButton: Story = {
+  args: {
+    titleLarge: 'About this Game',
+    description: longDescription,
+    sticker: [
+      {
+        label: 'Access Gated',
+        withIcon: <AlertTriangle data-icon="alert-triangle" />
+      },
+      {
+        label: 'On-chain assets available',
+        withIcon: <Info data-icon="info" />
+      }
+    ],
+    buttonLink: {
+      expanded: false,
+      onClick: () => console.log('Button clicked')
+    },
+    i18n: {
+      showMore: 'Show more',
+      showLess: 'Show less'
     }
-  />
-)
+  },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement)
+    await new Promise((resolve) => setTimeout(resolve, 2000))
+
+    await userEvent.click(canvas.getByTestId('show-more-button'), {
+      delay: 500
+    })
+
+    await expect(canvas.getByTestId('show-more-button')).toHaveStyle({
+      display: 'flex'
+    })
+
+    await new Promise((resolve) => setTimeout(resolve, 2000))
+    await userEvent.click(canvas.getByTestId('show-more-button'), {
+      delay: 500
+    })
+    await expect(canvas.getByTestId('show-more-button')).toHaveStyle({
+      display: 'flex'
+    })
```

Comment:
hmm I wonder if we can check whether the full description is shown here before and after the click :thinking: